extensions/websockets-next/server/deployment/src/main/java/io/quarkus/websockets/next/deployment/WebSocketServerProcessor.java

```diff
@@ -434,13 +474,102 @@ private String generateEndpoint(WebSocketEndpointBuildItem endpoint,
             onCloseExecutionModel.returnValue(onCloseExecutionModel.load(callback.executionModel));
         }
 
+        generateOnError(endpointCreator, endpoint, argumentProviders, transformedAnnotations, globalErrorHandlers, index);
+
         endpointCreator.close();
         return generatedName.replace('/', '.');
     }
 
+    private void generateOnError(ClassCreator endpointCreator, WebSocketEndpointBuildItem endpoint,
```

Comment:
I'd love to have some Javadoc showing an example of what this generated

Reply:
I can add something to the javadoc of the `generateEndpoint()`  method so that it can be seen in context.